lib/ach_client/version.rb

```diff
@@ -1,4 +1,4 @@
 module AchClient
   # Increment this when changes are published
-  VERSION = '3.1.0'
+  VERSION = '3.2.0'
```

Comment:
I think this should be major version bump to 4.0.0 since changing ruby compatibility is breaking change